### ISSUE TITLE
Action view - Grid Settings reactivity error

### ISSUE THREAD

**Issue by aaronjudd** [reactioncommerce/reaction#3404]
Opening the `Grid Settings` panel from the product grid and several performance issues appear:
These are experience items, hard to capture in screenshots, video, so you'll need to reproduce:

**Actual:**

1 - select product
2- Panel opens VERY sluggish
3- clicking alternate layouts provides no response (updates in console occur)
4 - grid isn't updated to new layout (but refreshing the page will show the correct it.

**Expected**

Reactive changes to grid should happen when layouts are selected.
Grid settings should be responsive when opening, selecting additional products.

```
Node: 6.11.1
NPM: 5.6.0
Meteor Node: 8.8.1
Meteor NPM: 5.4.2
Reaction CLI: 0.24.2
Reaction: 1.6.1
Reaction branch: master
Docker: 17.09.0-ce
```

Reproduced in #3368 as well.

**Comment by rymorgan** [comment]
Maybe related, but while testing last night I noticed panels opening with severe sluggishness. I'd guess that it's more of a performance issue vs an animation issue as it didn't it's taking much longer than it used to toggle open panels. 

![wkbyvhyipv 1](https://user-images.githubusercontent.com/20409254/33960079-6a7e9b0e-dffe-11e7-9d52-74a41e82ea01.gif)

**Comment by aaronjudd** [comment]
@rymorgan yes, that's the same sluggish behavior I'm seeing.

**Comment by zenweasel** [comment]
The most extreme example of this is the localization panel where trying to enable/disable a currency can take 5-10 to enable/disable. Some people from the community have commented on this as well. The problems does seem to be around that toggle slider?

**Comment by spencern** [comment]
@Akarshit what's the status on this issue?

**Comment by Akarshit** [comment]
@spencern The PR #3463 fixes the performance issues, I tested with ~200 products, but that doesn't make it super fast. The majority of the UI is still re-rendered when the Action Panel is opened. All the product thumbnails, navbar, action panel items are re-rendered and I don't see a way of preventing that from happening.

Any changes in the Grid Settings are reflected on the layouts(reactively) now. I think the PR will solve this problem(mostly by the 80/20 rule).

**Comment by zenweasel** [comment]
Closed via #3463
